storage/HashTable.hpp

```diff
@@ -1429,6 +1492,11 @@ HashTablePutResult HashTable<ValueT, resizable, serializable, force_key_copy, al
                                        variable_size,
                                        (*functor)(*accessor),
                                        using_prealloc ? &prealloc_state : nullptr);
+            // Insert into bloom filter, if enabled.
+            if (has_build_side_bloom_filter_) {
+              this->build_bloom_filter_->insert(static_cast<const std::uint8_t *>(key.getDataPtr()),
```

Comment:
Thanks. That's a good point. I guess inserting into a local bloom filter and doing a union with the shared bloom filter with locking may solve this problem. Sure, I will stop by today in the afternoon to discuss this in detail.